src/Scripting/Core/ScriptOptions.cs

```diff
@@ -54,6 +53,7 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
                 "System.Reflection.Extensions",
                 "System.Reflection.Primitives",
                 "System.Runtime",
+                "System.Runtime.Extensions",
```

Comment:
I'm going to try to change the tests to use APIs that are already available in the surface area.

---
In reply to: [110973145](https://github.com/dotnet/roslyn/pull/18471#discussion_r110973145) [](ancestors = 110973145,110970347)